sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeComposeFragment.kt

```diff
@@ -25,8 +35,31 @@ class HomeComposeFragment : BaseComposeFragment() {
             )
         }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        requestPermissions(
+            Manifest.permission.ACCESS_FINE_LOCATION,
+            Manifest.permission.CAMERA
```

Comment:
(Optional) Use static import
```suggestion
            ACCESS_FINE_LOCATION,
            CAMERA
```